server/models/db.json

```diff
@@ -0,0 +1 @@
-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"},{"id":"a9de8df5-97e0-4993-a5ec-fdfb935367f0","createdOn":"2019-01-06T22:37:41.716Z","location":"CCHub, Yaba, Lagos.","images":[],"topic":"Python Web Developers Meetup","body":"True Pythonistas Meetup!","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"},{"id":"70ba574f-6e88-4935-a30f-ed5e348aafee","createdOn":"2019-01-06T22:39:05.596Z","location":"CCHub, Yaba, Lagos.","images":[],"topic":"Python Web Developers Meetup","body":"True Pythonistas Meetup!","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"},{"id":"647b9c98-f650-4467-abad-6cd3bec13e5f","createdOn":"2019-01-06T22:39:21.781Z","location":"CCHub, Yaba, Lagos.","images":[],"topic":"Python Web Developers Meetup","body":"True Pythonistas Meetup!","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"},{"id":"b4fbd521-5028-4fa2-9f32-1889d7a7033d","createdOn":"2019-01-06T22:40:16.030Z","location":"CCHub, Yaba, Lagos.","images":[],"topic":"Python Web Developers Meetup","body":"True Pythonistas Meetup!","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"},{"id":"321730b7-45a7-4947-86c6-268c545b3048","createdOn":"2019-01-06T23:16:37.280Z","location":"CCHub, Yaba, Lagos.","images":[],"topic":"Python Web Developers Meetup","body":"","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"},{"id":"0fe3a505-87e3-446b-a01e-47617d982b0d","createdOn":"2019-01-07T04:47:23.626Z","location":"CCHub, Yaba, Lagos.","images":[],"topic":"Python Web Developers Meetup","body":"True Pythonistas Meetup!","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"},{"id":"5804dfc8-9543-48a3-961c-6fa02f7d108a","createdOn":"2019-01-07T05:03:22.477Z","location":"CCHub, Yaba, Lagos.","images":[],"topic":"Python Web Developers Meetup","body":"True Pythonistas Meetup!","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"},{"id":"25165214-4757-4522-8e20-7b434c0d4867","createdOn":"2019-01-07T05:04:40.341Z","location":"CCHub, Yaba, Lagos.","images":[],"topic":"Python Web Developers Meetup","body":"True Pythonistas Meetup!","happeningOn":"2018-12-23T12:00:00.511Z","tags":["Programming","Python","Web"],"convener":"James Richard"}],"rsvps":[{"id":"cb9e78d3-853c-44b1-af7b-616411a8edcd","meetup":"6ecd62b7-e473-40e0-a6b1-eb9440b31b24","topic":"German Language Learners Group","response":"yes"},{"id":"b51b859a-ce4f-46b2-bc01-e5e98f00f81b","meetup":"77606213-7fd0-4d74-a8b6-c05c789b0f72","topic":"German Language Learners Group","response":"yes"},{"id":"8da0e5b7-3443-4db6-8329-9341a0f6bf7c","meetup":"77606213-7fd0-4d74-a8b6-c05c789b0f72","topic":"German Language Learners Group","response":"yes"}]}
```

Comment:
A very good implementation.
But you are to use in-memory storage.
Please replace the `db.json` file with a `db.js` file and define your data structure the same way.

Reply:
Noted. Thanks